VTEX - Ad Network API.json

```diff
@@ -61,6 +61,25 @@
                         "$ref": "#/components/parameters/hideUnavailableItems"
                     }
                 ],
+                "servers": [
+                    {
+                        "url": "https://{accountName}.{environment}.com.br/api/io/_v/api/intelligent-search",
+                        "description": "VTEX Intelligent Search server URL.",
```

Comment:
Should be something like: `VTEX Ad Network server URL integrated with Intelligent Search.`